types/es-abstract/es2015.d.ts

```diff
@@ -71,10 +69,21 @@ interface ES2015 extends Omit<typeof ES5, 'CheckObjectCoercible' | 'ToPrimitive'
     CompletePropertyDescriptor<D extends ES5.PropertyDescriptor>(
         Desc: D & ThisType<any>,
```

Comment:
i'd prefer all these any's changed to unknown's also, but that can be in a separate PR too

Reply:
I intend to do it in <https://github.com/DefinitelyTyped/DefinitelyTyped/pull/44805>.